examples/add_public_raster_dataset.ipynb

```diff
@@ -187,6 +178,56 @@
     "meta_data"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'id': None,\n",
+       " 'name': 'Land Cover',\n",
+       " 'description': 'Land Cover',\n",
+       " 'sourceOperator': 'GdalSource',\n",
+       " 'symbology': {'type': 'raster',\n",
+       "  'colorizer': {'type': 'linearGradient',\n",
+       "   'breakpoints': [{'value': 0, 'color': (0, 0, 255, 255)},\n",
+       "    {'value': 8, 'color': (0, 255, 0, 255)},\n",
+       "    {'value': 16, 'color': (255, 0, 0, 255)}],\n",
+       "   'noDataColor': (0, 0, 0, 0),\n",
+       "   'defaultColor': (0, 0, 0, 0)},\n",
+       "  'opacity': 1.0},\n",
+       " 'provenance': None}"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "add_dataset_properties = ge.AddDataset( \n",
+    "    name = \"Land Cover\",\n",
+    "    source_operator = \"GdalSource\",\n",
+    "    description = \"Land Cover\",\n",
+    "    symbology = ge.RasterSymbology(\n",
+    "      opacity = 1.0,\n",
```

Comment:
the task said you should also be able to specify the provenance info.

Reply:
[c68f527](https://github.com/geo-engine/geoengine-python/pull/94/commits/c68f527e25a02008e31d51e131036eacf4b8e6f9)